test/test_integration.py

```diff
@@ -494,6 +494,43 @@ def update_pathname(n_clicks, current_pathname):
         self.wait_for_text_to_equal('#test-hash', '')
         self.snapshot('link -- /test/pathname/a?queryA=valueA')
 
+    def test_link_scroll(self):
+        app = dash.Dash(__name__)
+        app.layout = html.Div([
+            dcc.Location(id='test-url', refresh=False),
+
+            html.Div(id='push-to-bottom', children=[], style={
+                'display': 'block',
+                'height': '200vh'
+            }),
+            html.Div(id='page-content'),
+            dcc.Link('Test link', href='/test-link', id='test-link')
+        ])
+
+        @app.callback(Output('page-content', 'children'),
+                       [Input('test-url', 'pathname')])
+        def display_page(pathname):
+            return 'You are on page {}'.format(pathname)
```

Comment:
Let's increment a `multiprocessing.Value` in here so that we can ensure that the callback is only fired a single time. 
Here's an example of incrementing `Value` in a callback:
https://github.com/plotly/dash-renderer/blob/70bb93e8f0151ddc7d0810f3df271036a069360a/tests/test_render.py#L458-L463

and further down, the assertion:
https://github.com/plotly/dash-renderer/blob/70bb93e8f0151ddc7d0810f3df271036a069360a/tests/test_render.py#L478-L484